config/coordinator.sample.json

```diff
@@ -13,34 +13,51 @@
     "UpdateIntervalMs": 2000,
     "Queries": [
       {
-        "Description": "Average request duration (old)",
+        "Description": "Average server request duration (old)",
         "Query": "rate(mattermost_http_request_duration_seconds_sum[1m])/rate(mattermost_http_request_duration_seconds_count[1m])",
         "Threshold": 0.1,
         "Alert": true
       },
       {
-        "Description": "Average request duration",
+        "Description": "Average server request duration",
         "Query": "sum(rate(mattermost_api_time_sum[1m]))/sum(rate(mattermost_api_time_count[1m]))",
         "Threshold": 0.1,
         "Alert": true
       },
       {
-        "Description": "Rate of HTTP 5xx client errors",
-        "Query": "sum(rate(loadtest_http_errors_total{status_code=~\"5..\"}[1m]))",
-        "Threshold": 0.01,
+        "Description": "Percentage of HTTP 5xx server errors",
+        "Query": "(sum(rate(mattermost_api_time_count{status_code=~\"5..\"}[1m]))/sum(rate(mattermost_api_time_count[1m])))*100",
+        "Threshold": 0.025,
         "Alert": true
       },
       {
-        "Description": "Rate of client timeouts",
-        "Query": "sum(rate(loadtest_http_timeouts_total[1m]))",
-        "Threshold": 0.01,
+        "Description": "Average client request duration",
+        "Query": "sum(rate(loadtest_http_request_time_sum[1m]))/sum(rate(loadtest_http_request_time_count[1m]))",
+        "Threshold": 0.1,
+        "Alert": true
+      },
+      {
+        "Description": "99th percentile of client request duration",
+        "Query": "histogram_quantile(0.99, sum(rate(loadtest_http_request_time_bucket[1m])) by (le))",
+        "Threshold": 2.0,
+        "Alert": true
+      },
+      {
+        "Description": "Percentage of HTTP 5xx client errors",
+        "Query": "(sum(rate(loadtest_http_errors_total{status_code=~\"5..\"}[1m]))/sum(rate(loadtest_http_request_time_count[1m])))*100",
+        "Threshold": 0.025,
+        "Alert": true
+      },
+      {
+        "Description": "Percentage of client timeouts",
+        "Query": "(sum(rate(loadtest_http_timeouts_total[1m]))/sum(rate(loadtest_http_request_time_count[1m]))) * 100",
+        "Threshold": 0.025,
         "Alert": true
       }
-    ]
   },
-  "NumUsersInc": 16,
-  "NumUsersDec": 16,
-  "RestTimeSec": 10,
+  "NumUsersInc": 8,
+  "NumUsersDec": 8,
+  "RestTimeSec": 2,
```

Comment:
Since we implemented https://github.com/mattermost/mattermost-load-test-ng/pull/328 it makes sense to keep the loop more active which helps in finding an equilibrium more quickly. Lowering this down to be equal to the query update time.